src/components/generators/view-components/string.vue

```diff
@@ -54,4 +97,38 @@ export default {
 .fa-external-link:hover {
   cursor: pointer;
 }
+
+.toast-container {
+  display: block;
+  max-width: 350px;
+  backface-visibility: hidden;
+  -webkit-backface-visibility: hidden;
+  background-clip: padding-box;
+  z-index: 1;
+  border-radius: 0.25rem;
+}
+
+.toast-container-top-center {
+  position: fixed;
+  min-height: 200px;
+  margin-top: 5px;
+  top: 0;
+}
+
+.toast-container .toast {
+    background-color: rgba(230, 242, 255, 0.85);
+    border-color: rgba(184, 218, 255, 0.85);
+    color: #004085;
```

Comment:
for colors, use the bootstrap utility classes, so it adheres to any custom theme as well

Reply:
used alert-info class for the div